addons/crm_iap_enrich/models/crm_lead.py

```diff
@@ -53,7 +59,7 @@ def iap_enrich(self, from_cron=False):
                         {'lead_ids': tuple(leads.ids)}, log_exceptions=False)
                     for lead in leads:
                         # If lead is lost, active == False, but is anyway removed from the search in the cron.
-                        if lead.probability == 100 or lead.iap_enrich_done:
+                        if lead.probability == 100 or lead.iap_enrich_done and not lead.active:
```

Comment:
Somehow this change is back.

Can you either remove it or explain why you do it?
And I mean, where it was asked in the task specifications?
I don't see any reference to the archive mechanism in there.